docker-compose.yml

```diff
@@ -28,18 +28,6 @@ services:
       - SECRET_KEY=dev
       - AUTHORIZATION_TOKEN=auth-token
       - GET_AUTHORIZATION_TOKEN=get-auth-token
-      - UWSGI_HTTP=0.0.0.0:8001
-      - UWSGI_MASTER=1
-      - UWSGI_STATIC_INDEX=index.html
-      - UWSGI_STATIC_MAP=/mensen/static=/static
-      - UWSGI_CALLABLE=application
-      - UWSGI_VACUUM=1
-      - UWSGI_STATIC_EXPIRES=/* 3600
-      - UWSGI_OFFLOAD_THREADS=1
-      - UWSGI_HARAKIRI=15
-      - UWSGI_DIE_ON_TERM=1
```

Comment:
deze missen in de nieuwe config. Maar waren misschien overbodig?

Reply:
Ik heb de volgenden toegevoegd:
- `static-expires`
- `offload-threads`
- `harakiri`

Volgens de docs [is die-on-term niet meer nodig](https://uwsgi-docs.readthedocs.io/en/latest/Options.html#die-on-term).